vscode-extension/src/extension.ts

```diff
@@ -812,4 +819,4 @@ async function shareAIConfig(
         }
       });
   }
-}
+}
```

Comment:
nit, newline. Do you have prettier running?

Reply:
my prettier settings were changed, I reset it to default settings and it worked like a charm